proto/bragi/common.proto

```diff
@@ -49,25 +49,33 @@ message ControlErrorAnnouncement {
   optional string message = 2;
 }
 
+message MatchTimeline {
+  repeated Match matches = 1;
```

Comment:
Btw is it good idea use Match used in feed for timeline purposes? The data size can be big in the future.

Reply:
It's used as a timeline message only. It means it will contain only details required by the timeline